src/documents/resources/index.html.md

```diff
@@ -40,7 +40,7 @@ There are a bunch of cool stuff in this [NodeJS](http://nodejs.org/) tool that w
 
 Now that you finished your element it's time to spread the word about it! You might not believe me but there are other people in the world that could benefit from it, so why not share it with them?
 
-Nowadays, [Bower](http://bower.io/) is the most popular package manager for front-end code, that's why we recommend you to use it.
+The best way to share a component is by making it available for use with one of the current JavaScript package managers. Currently most web components libraries have great support for [Bower](http://bower.io/), that's why we recommend you to use it. Keep in mind that Bower is not the only package manager for front-end code. [npm](https://www.npmjs.com/) is becoming extremely popular for managing JavaScript pacakges, and it's possible that it will be able to completely replace Bower in the future. For simplicity's sake, we'll stick with Bower for now.
```

Comment:
- Should we include JSPM ? 
- "most web components libraries" should be "most web component libraries"